entry_types/scrolled/package/src/frontend/Entry.js

```diff
@@ -18,13 +18,24 @@ export default withInlineEditingDecorator('EntryDecorator', function Entry(props
   const [scrollTargetSectionIndex, setScrollTargetSectionIndex] = useState(null);
 
   const entryStructure = useEntryStructure();
-
   useSectionChangeEvents(entryStructure, currentSectionIndex);
 
+  let updateChapterSlug = (slug) => {
+    history.replaceState(null, null, '#'+slug);
```

Comment:
**issue**: History is still used without feature detection

Also linter still complains that this should be `window.history`.